common/adaptive-bitrate.md

```diff
@@ -0,0 +1,100 @@
+Adaptive bitrate (ABR) refers to features that enable dynamic adjustments to video quality—to optimise for end-user experience under diverse network conditions. ABR ensures that every participant is able to consume the highest possible quality video in conferencing or streaming use-cases, based on their bandwidth constraints.
+
+In addition to network, ABR can also optimise for the right video quality based on the size of the video element. For example, a video call running on active speaker layout has larger video tiles that require higher quality video track. These adjustments can be made dynamically with adaptive bitrate.
```

Comment:
> or example, a video call running on active speaker layout has larger video tiles that require higher quality video track 
Maybe some diagrams would help here. Is it clear to people what active speaker layout is?

Reply:
i've added a demo video for this